src/test/java/com/acoustic/SpringPolandSalaryCalculator/calculator/SalaryCalculatorTest.java

```diff
@@ -0,0 +1,109 @@
+package com.acoustic.SpringPolandSalaryCalculator.calculator;
+
+import java.math.BigDecimal;
+import java.math.RoundingMode;
+import java.util.LinkedHashMap;
+import java.util.Map;
+import java.util.function.UnaryOperator;
+
+import org.springframework.stereotype.Component;
+
+import com.acoustic.SpringPolandSalaryCalculator.rates.RatesConfigurationPropertiesTest;
+
+
+@Component
+public class SalaryCalculatorTest {
```

Comment:
What is going on in this test?

Reply:
It is an helper class to create the response body to compare with the response of the salaryCalculator method.
I have renamed it to SalaryCalculatorResponse.